rs/sns/cli/src/health.rs

```diff
@@ -57,11 +68,16 @@ impl TableRow for SnsHealthInfo {
         let low_cycles = self
             .cycles
             .iter()
-            .filter(|(cycles, _)| (*cycles as f64) < 10.0 * 1000.0 * 1000.0 * 1000.0 * 1000.0)
+            .filter(|(cycles, _)| (cycles.cycles as f64) < CYCLES_THRESHOLD_TC * TC)
             .map(|(cycles, canister_type)| {
                 format!(
-                    "{canister_type}: ({:.2} TC)",
-                    *cycles as f64 / 1000.0 / 1000.0 / 1000.0 / 1000.0
+                    "{canister_type}: ({:.2} TC{frozen})",
+                    cycles.cycles as f64 / TC,
+                    frozen = if (cycles.freezing_threshold as u128) > cycles.cycles {
```

Comment:
Nit. Such a strange way to write this condition. 

```suggestion
                    frozen = if cycles.cycles < cycles.freezing_threshold as u128 {
```

Reply:
Done!